api/src/controllers/client/options.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright 2022 Touca, Inc. Subject to Apache-2.0 License.
 
+import { WorkflowOptions } from '@touca/api-schema'
```

Comment:
Unrelated improvement: Defined this in `schema.yaml` so we can use it in route documentation.